### DISCUSSION TITLE
idea for an extension

### DISCUSSION THREAD

**Discussion by Vainstains** [ToniMacaroni/SaberFactory#67] [Ideas]
would it be possible to create an extension along the lines of SaberFactory.Sharp, so that saberfactory can run full scripts not just events? i have an idea for a saber where it is all black, and there is a single ring of color that gets longer down the blade the harder you swing, or possibly a saber EQ. the reason it would be an extension is because i get that you could do potentially bad things with IO, so if it is an extension then you would have to actually download it and get a warning of caution.

**Comment by ToniMacaroni** [comment]
Yeah I always had an uneazy feeling about user scripts in mods.
But if users agree to the risk or if it's your own saber I think it should be ok to at least give the possibility.
